src/Components/ColorPallete/ColorPallete.jsx

```diff
@@ -0,0 +1,33 @@
+import { useUserContext } from "../../Context";
+import { updateNotes } from "../../Services";
+
+export function ColorPallete({ noteDetails }) {
+  const { userDispatch } = useUserContext();
+  const colorPallete = [
```

Comment:
can extract this to a constants file

Reply:
Thanks for suggestion,  I will do it later